### ISSUE TITLE
stayman is not game forcing

### ISSUE THREAD

**Issue by Muppet1** [EdwardPiwowar/BBA#394]
![image](https://github.com/EdwardPiwowar/BBA/assets/11272970/ab5144fd-8a68-412d-89b2-140e4e611973)
0012D0B086AAE78C035D66F0EF25

if i want to game force I just use smolen.

if i want to just play in a major I bid like this.

**Comment by ThorvaldAagaard** [comment]
This is OK for GIB, but I think SAYC use garbage Stayman like suggested

**Comment by Muppet1** [comment]
2 is “non-forcing” Stayman, meaning that the bidding may stop in two of a suit. 


from sayc pdf.

**Comment by ThorvaldAagaard** [comment]
Yes, and as you mention it is related to the use of Smolen, that GIB is not using.

**Comment by Muppet1** [comment]
![image](https://github.com/EdwardPiwowar/BBA/assets/11272970/dc2b2e3a-bc97-4403-a837-1f87680709a7)
0E6D2FC84190BCE750918D16E7B1

please let me play 2 spades.

**Comment by Muppet1** [comment]
![image](https://github.com/EdwardPiwowar/BBA/assets/11272970/c9c7fe36-87d3-40dd-bb68-919c4cabe374)
040293172DA9785FCD67B6F2DEA5

continuing the stayman being an invite plus problem.

Here South wants to use stayman and not invite.  I'll just keep posting these as they happen which is all the time.

**Comment by EdwardPiwowar** [comment]
1. Seems ok now
![image](https://github.com/EdwardPiwowar/BBA/assets/150140020/f559d9ca-7c49-4f36-aa8a-7a02a658450e)
2. I don't know if I can pass on 2S 8+
3. Garbage Stayman isn't implemented yet.
